src/main/java/seedu/address/MainApp.java

```diff
@@ -67,14 +67,18 @@ public void init() throws Exception {
             config.getUserPrefsFilePath()
         );
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        ActivityBookStorage activityBookStorage = new JsonActivityBookStorage(userPrefs.getActivityBookFilePath());
+
         InternalStateStorage internalStateStorage = new JsonInternalStateStorage(
             userPrefs.getInternalStateFilePath()
         );
         InternalState internalState = initState(internalStateStorage);
+
+        ActivityBookStorage activityBookStorage = new JsonActivityBookStorage(userPrefs.getActivityBookFilePath());
```

Comment:
This part was fine, the initialization of saved books do not affect the internal state, but OK.